crates/pyo3/ast_grep_pyo3/__init__.py

```diff
@@ -31,7 +33,7 @@ class Rule(RuleWithoutNot, TypedDict("Not", {"not": "Rule"}, total=False)):
     pass
 
 # Relational Rule Related
-StopBy = Literal["neighbor"] | Literal["end"] | Rule
+StopBy = Union[Literal["neighbor"], Literal["end"], Rule]
```

Comment:
Since this is not a type annotation, PEP 563 has no effect on this, so we can only use the `Union`.

Reply:
Thanks!